components/content-picker/PickedItem.js

```diff
@@ -8,7 +8,7 @@ import { __ } from '@wordpress/i18n';
 import { useSelect } from '@wordpress/data';
 import { useEffect } from '@wordpress/element';
 
-const DragHandle = (props) => (
+export const DragHandle = (props) => (
```

Comment:
Let's move this into its own component while we are at it. Makes it easier to maintain

Reply:
No I don't think so. If we ever expand it in the future maybe but not right now.